contracts/PetPark.sol

```diff
@@ -1,7 +1,105 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    enum AnimalType {
+        NotValid,
+        Fish,
+        Cat,
+        Dog,
+        Rabbit,
+        Parrot
+    }
+    enum Gender {
+        Male,
+        Female
+    }
+
+    struct RenterInfo {
+        uint8 age;
+        Gender gender;
+    }
+    mapping(AnimalType => uint) public petsInShelter;
+    mapping(address => RenterInfo) public renters;
+    mapping(address => AnimalType) public rentedPet;
+    address public owner;
+
+    event Added(uint8 animalType, uint8 animalCount);
```

Comment:
You can define it as `event Added(AnimalType animalType, uint8 animalCount);` since you have declared enums